index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog Home</title>
+  </head>
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="">
+      <a href="index.html"><h1>Adoptadog</h1></a>
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <aside>
+        <!--icon for checkout-->
+        <!--total, ecommerce feature-->
+        <a class="#" href="checkout.html">checkout</a>
+      </aside>
+    </header>
+    <nav>
+      <a class="#" href="index.html"><h6>Home</h6></a>
+      <a class="#" href="gallery.html"><h6>Dogs</h6></a>
+      <a class="#" href="blog.html"><h6>Blog</h6></a>
+    </nav>
+    <div>
+      <div class="#">
+        <figure>
+          <img src="images/home-intro.jpg" alt="">
+        </figure>
+        <section>
+          <h3>Adopt A Dog</h3>
+          <p>Lorem ipsum dolor amet fashion axe beard hashtag lyft, keytar kickstarter franzen tattooed disrupt ipsum banjo tacos ennui roof party gentrify. Dreamcatcher locavore blog literally, waistcoat kombucha narwhal et food truck hoodie yuccie pariatur stumptown. Woke aliquip glossier butcher. Master cleanse food truck in street art. Farm-to-table authentic godard dolore cillum, slow-carb celiac. Pickled copper mug ad cred etsy swag. Excepteur activated charcoal lyft, tattooed small batch kickstarter disrupt tousled meh before they sold out direct trade exercitation seitan gastropub.</p>
+        </section>
+      </div>
+      <div class="#">
+        <section class="#">
+          <figure>
+            <img src="images/murphy-card.jpg" alt="">
+          </figure>
+          <h5>Murphy</h5>
+          <p><b>Cost to Adopt:</b>$123.45</p>
```

Comment:
Instead of using the b tag, use strong. You'll need to style it using CSS, but it's a better way to go. Alternatively you could use a span, but strong is a better choice.